lib/rules/no-sequences.js

```diff
@@ -26,14 +26,22 @@ module.exports = {
             url: "https://eslint.org/docs/rules/no-sequences"
         },
 
-        schema: [],
+        schema: [{
+            properties: {
+                allowInParentheses: {
+                    type: "boolean",
+                    default: false
```

Comment:
should defaults to `true`. - we can change the default option only in a major release.

Reply:
👍 the option name make sense to me!